sdk/ml/azure-ai-ml/azure/ai/ml/dsl/_pipeline_decorator.py

```diff
@@ -57,6 +59,7 @@ def pipeline(
     description: str = None,
     experiment_name: str = None,
     tags: Dict[str, str] = None,
+    non_pipeline_parameters: List[str] = None,
```

Comment:
we need to hide this in argv. This is an internal feature

Reply:
Hide this parameter in kwargs